growth/too/tasks/ztf_client.py

```diff
@@ -89,3 +92,73 @@ def ztf_references():
                          reference_filter_ids=rows['fid'].tolist(),
                          reference_filter_mags=rows['maglimit'].tolist()))
     models.db.session.commit()
+
+
+@celery.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_depot(start_time=None, end_time=None):
+    """
+    ZTF depot reader, ingesting information about images from
+    all program ids (including program_id = 1) based on the
+    nightly summary. This supplements what is available from
+    the TAP interface, where information about public images
+    is not available.
+
+    Parameters:
+        start_time (astropy.Time): start time of request.
+        end_time (astropy.Time): end time of request.
+    """
+
+    if start_time is None:
+        start_time = time.Time.now() - time.TimeDelta(1.0*u.day)
+    if end_time is None:
+        end_time = time.Time.now()
+
+    depotdir = 'https://ztfweb.ipac.caltech.edu/ztf/depot'
+
+    mjds = np.arange(np.floor(start_time.mjd), np.ceil(end_time.mjd))
+    print(mjds)
```

Comment:
Please remove this print statement. Use the logging subsystem if there is useful telemetry that you want to record here.